src/MultiStopGradient.js

```diff
@@ -22,23 +24,10 @@ const putInRange = (ranges, val) => {
 
 
 const lineHeight = 24
-const tickLabelStyles = {
-  leftLabel: {
-    right: `20px`,
-    paddingRight: `5px`,
-    position: `absolute`,
-    marginTop: `-${lineHeight / 2}px`
-  },
-  rightLabel: {
-    left: `20px`,
-    paddingLeft: `5px`,
-    position: `absolute`,
-    marginTop: `-${lineHeight / 2}px`
-  }
-}
+
 const Tick = ({value, colour, top, position}) =>
-  <div style={{position: `absolute`, height: `2px`, width: `20px`, background: colour, top: `${top + lineHeight}px`}}>
-    <div style={(position === 'left' ? tickLabelStyles.leftLabel : tickLabelStyles.rightLabel)}>
+  <div className={styles.tick} style={{background: colour, top: `${top + lineHeight}px`}}>
+    <div className={styles[position]}>
```

Comment:
I wasn’t aware that a CSS import was a JS object with classes as field names. Very cool!